apps/common/components/Pagination.tsx

```diff
@@ -0,0 +1,62 @@
+import ReactPaginate from 'react-paginate';
+import IconPaginationArrow from '@common/icons/IconPaginationArrow';
+
+import type {ReactElement} from 'react';
+
+type TProps = {
+	range: [from: number, to: number];
+	pageCount: number;
+	numberOfItems: number;
+	onPageChange: (selectedItem: {
+		selected: number;
+	}) => void;
+}
+
+export function Pagination(props: TProps): ReactElement {
+	const {range: [from, to], pageCount, numberOfItems, onPageChange} = props;
+
+	return (
+		<>
+			<div className={'flex flex-1 justify-between sm:hidden'}>
+				<a
+					href={'#'}
+					className={'border-gray-300 text-gray-700 hover:bg-gray-50 relative inline-flex items-center rounded-md border  px-4 py-2 text-sm font-medium'}
```

Comment:
There was an extra space in the class name below this one, so refactored both with https://github.com/yearn/yearn.fi/pull/248/commits/aea45ed44c9a59467609f68b2c8367caa5df8f73 instead of committing your suggestion

Reply:
Oh, just noticed you've spotted that one too https://github.com/yearn/yearn.fi/pull/248#discussion_r1251660571 😅